base-notebook/Dockerfile

```diff
@@ -70,8 +70,10 @@ RUN sed -re "s/c.ServerApp/c.NotebookApp/g" \
 # This healtcheck works well for `lab`, `notebook`, `nbclassic`, `server` and `retro` jupyter commands
 # https://github.com/jupyter/docker-stacks/issues/915#issuecomment-1068528799
 HEALTHCHECK  --interval=5s --timeout=3s --start-period=5s --retries=3 \
-    CMD wget -O- --no-verbose --tries=1 --no-check-certificate \
-    http${GEN_CERT:+s}://localhost:${JUPYTER_PORT}${JUPYTERHUB_SERVICE_PREFIX:-/}api || exit 1
+    CMD bash -c 'set -o pipefail; \
```

Comment:
This is getting out of hand! 😂 

Let's write a simple python script, please 😄

Reply:
yeah I know =)) adding a simple bash option seems to resolve the issue, so I thought I'll give it a shot lol. alright, I'll try to work on the python script

Reply:
Anyway, when I run `make build/base-notebook` followed by `make test/base-notebook`, the test should run on the image I have locally, right?

Reply:
Actually, never mind. I got strange behavior, apparently because I forgot to rebuild the image locally

Reply:
> Anyway, when I run `make build/base-notebook` followed by `make test/base-notebook`, the test should run on the image I have locally, right?

Yes, but you need to build all parent images as well - in this case, if you want to have the latest version, you should first build `make build/docker-stacks-foundation`.